templates/candidates-single.html

```diff
@@ -66,14 +66,14 @@ <h1 class="entity__name">{{ name }}</h1>
         <div class="entity__election">
           <span class="entity__term__label">More information on this election</span>
           <a
-            class="button card card--feature card--full card--neutral"
+            class="button card card--horizontal card--full-bleed card--neutral"
             href="{{ election_url(context(), cycle) }}"
             >
             <div class="card__image__container">
               <img class="card__image icon--complex" src="/static/img/i-elections--primary.svg" alt="Icon representing elections">
             </div>
             <div class="card__content">
-              <span class="t-sans t-block">{{cycle}} {{office_full}}</span>
+              <h5 class="u-no-margin t-upper t-sans">{{cycle}} {% if office == 'P' %} Presidential election {% else %} {{office_full}} {% endif %}</h5>
```

Comment:
Yeah, that's the idea. Because presidential elections don't have a district, the callout box was just looking weird with just "2012 President" in it. Also, I think saying "Presidential election" is more natural than "President election", even though "House election" and "Senate election" are both natural to say.